public/components/custom_panels/panel_modules/visualization_container/visualization_container.tsx

```diff
@@ -25,6 +25,7 @@ import {
   EuiToolTip,
 } from '@elastic/eui';
 import React, { useEffect, useMemo, useState } from 'react';
+import _ from 'lodash';
```

Comment:
Is this import being used anywhere?

Reply:
it's fine here but let's align our usage gradually to import only used functions

Reply:
I'm not sure, I just rearranged this import as lint error was thrown during commit. I didn't import it